src/i18n.rs

```diff
@@ -117,60 +113,59 @@ impl HelperDef for TeamHelper {
         h: &Helper<'reg, 'rc>,
         _: &'reg Handlebars,
         context: &'rc Context,
-        rcx: &mut RenderContext<'reg>,
+        rcx: &mut RenderContext<'reg, 'rc>,
         out: &mut dyn Output,
     ) -> HelperResult {
-        let name = if let Some(name) = h.param(0) {
-            name
-        } else {
+        let Some(name) = h.param(0) else {
             return Err(RenderError::new(
                 "{{team-text}} must have at least two parameters",
             ));
         };
-        let name = if let Some(name) = name.path() {
-            name
-        } else {
+        let Some(name) = name.relative_path() else {
             return Err(RenderError::new(
                 "{{team-text}} takes only identifier parameters",
             ));
         };
 
-        let id = if let Some(id) = h.param(1) {
-            id
-        } else {
+        let Some(id) = h.param(1) else {
             return Err(RenderError::new(
                 "{{team-text}} must have at least two parameters",
             ));
         };
-        let id = if let Some(id) = id.path() {
-            id
-        } else {
+        let Some(id) = id.relative_path() else {
             return Err(RenderError::new(
                 "{{team-text}} takes only identifier parameters",
             ));
         };
         let team = rcx
-            .evaluate_in_block_context(name)?
-            .ok_or_else(|| RenderError::new(format!("Cannot find team {}", name)))?;
+            .evaluate(context, name)
+            .map_err(|e| RenderError::from_error(&format!("Cannot find team {}", name), e))?;
```

Comment:
This part is basically guesswork on my part. I have zero experience with handlebars, so these helper idioms are unknown to me. `evaluate_in_block_context` doesn't exist anymore. I found no hints about this in the changelog of handlebars and `evaluate` isn't even documented in the current version :disappointed: .

I'm guessing these teams are related to the governance pages though, and they seem to work fine (after some debugging).